### PR TITLE
Updated schemas

### PR DESCRIPTION
**Pull request by JettTech**

### CODE REVIEW

File: agent-1.key [hunk 1]
```diff
@@ -0,0 +1 @@
...4NiwyMDMsNCwxMjAsMTE2LDEzOSwyMzYsMjAsMTMwLDAsMTg4LDE2MywyNDIsMjMwLDIzOSw1OV0sImNpcGhlciI6WzE4OSwyMzksMjQ4LDE4MCw1Nyw2OCwxNzAsMjQyLDE2MiwzOCwyNDUsMjQ2LDEwMyw2LDE4MCwxODgsMjksNzQsMTY5LDE1NCwyLDE1MSw1OSwxODUsMjE3LDIzNiwxODgsMTY2LDMwLDcsMjMxLDc3LDQsMjUsMTc0LDExMCwxNjYsMTI4LDExMiwxMzMsMTAzLDIxOCwxMyw0NiwyMjAsMTgsMTM4LDIxOCwxMzEsODAsMjIwLDIyOCwxODksMTQ1LDIwMyw2NSwyNDEsMTQ4LDE2MCwxMTUsMjM5LDU3LDEwNSw2LDExMCwxOSwyMDYsNTQsMTk2LDIsNDMsMTAxLDIxNCw1OCwzMywxNDQsMTk5LDE1MCwyMzQsODgsMjA1LDEwNCwxMzMsMTQ1LDExMywyMzksOTIsMjE1LDIzOSw0Nyw5MiwwLDQ0LDg0LDcxLDE1OCw0MywxNzUsMjQ0LDQyLDgzLDI0MiwxOTIsMTQwLDIwMSw5MywxMjIsMTc4LDI1NCwxMTksMTI5LDU0LDE4MiwyMDcsMTMsMTEyLDExMCwxMTEsMTA4LDk0XX0="},"root_seed":{"blob_type":"Seed","seed_type":"OneShot","hint":"","data":"eyJzYWx0IjpbMyw0MSw4Niw1OCw5OCw0Nyw4LDM5LDc4LDEwMywyNTEsMTI5LDEzNyw0NSwyNTAsNjBdLCJub25jZSI6WzEzOCwxMzEsODUsMzEsMTAwLDYsMTk1LDM5LDE2MywxMTEsMjU0LDExOSwyMjAsMTMsNzksMTEyLDYxLDYxLDE5NywyMjUsMTk1LDI1Miw4NCw5Ml0sImNpcGhlciI6WzEyMCwyNTQsMjYsMjI3LDEzOCwyMjIsODAsMjEzLDcxLDIxOCwxNzMsNjIsMjE5LDMsMTUxLDI0NiwxNDcsMTg4LDk5LDEzLDExOSwyMTAsOTMsNjYsMTk3LDIxMSwxNzYsMzYsMjcsMTI1LDU3LDYyLDE5OCwyMjksMTcsODMsNiwxMjUsMjAxLDI1MywyMTAsMjQ4LDU4LDYyLDEyMyw3LDE0NCwxNTddfQ=="}}}
```
Comment (robbiecarlton) [inline agent-1.key]: Do we want to commit this?

File: conductor-config.toml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+bridges = []
```
Comment (robbiecarlton) [inline conductor-config.toml]: Do we want to commit this?

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
Everything looks good, except I think we should not include the conductor set up stuff in here. That's part of the next ticket (https://www.pivotaltracker.com/story/show/171367992) and will probably live in the outer repo, not in here. 

So I think we remove `agent-1.key` and `conductor-config-toml` and line 31 of `package.json`.

Other than that good to merge

**Comment by JettTech** [comment]
@robbiecarlton - I'm fine moving this out considering this will most likely merge into the DNA repo soon.  That said, I would like to have a branch that has this included just for local testing setup, to allow for coordinated asynchronous testing, but I'm fine moving that out of this branch.  Thoughts?

**Pull request merged**